AppCenterDistribute/AppCenterDistribute/MSACDistribute.m

```diff
@@ -144,6 +144,12 @@ - (instancetype)init {
 
     // Init the distribute info tracker.
     _distributeInfoTracker = [[MSACDistributeInfoTracker alloc] init];
+
+    // In case we never finished removing this key. Which will result in
+    // update flow never gets processed again
```

Comment:
```suggestion
    /*
     * In case we never finished removing this key. Which will result in
     * update flow never gets processed again.
     */
```

Reply:
Updated, thanks!